uint8 group_id
uint8 param_id
uint8 num_params
string param_name

uint8 TYPE_INT = 1
uint8 TYPE_FLOAT = 2
uint8 type

int32 value_int
float32 value_float
